### ISSUE TITLE
Not working in GNOME 43

### ISSUE THREAD

**Issue by D3vil0p3r** [vinceliuice/Graphite-gtk-theme#85]
Hello,
I always used this wonderful theme, but now that I upgraded GNOME to 43, it is not working anymore:
![image](https://user-images.githubusercontent.com/83867734/199968865-f3eac07a-f3ba-4a08-b21f-6df51f673dd0.png)

Is there the chance to make it compatible with GNOME 43?

**Comment by vinceliuice** [comment]
Run: `./install.sh -l`

**Comment by D3vil0p3r** [comment]
> Run: `./install.sh -l`

Thank you for the answer. It seems to work when I install it with the command provided on the session of the current user. If I try to install it system-wide, so `sudo ./install.sh -l`, where the theme will be installed in `/usr/share/themes`, it seems to not work.

**Comment by vinceliuice** [comment]
Yes it can not work for sudo, because this fix for libadwaita is just link gtk-4.0 theme to user's gtk-4.0 config folder, when you run sudo the user is root, so it'll not work

**Comment by D3vil0p3r** [comment]
> Nope it can not work for sudo, because this fix for libadwaita is just link gtk-4.0 theme to user's gtk-4.0 config folder, when you run sudo the user is root, so it'll not work

Uhm now I tried to generate the Graphite-Dark-compact theme on my local folder that generated the links to gtk-4.0 local folder. After this generation, I moved Graphite-Dark-compact from local `$HOME/.themes` folder to `/usr/share/themes`, I deleted all the libadwaita local links and then I just simulated the script linking as:
```
mkdir -p "${HOME}/.config/gtk-4.0"
ln -sf "/usr/share/themes/Graphite-Dark-compact/gtk-4.0/assets" "${HOME}/.config/gtk-4.0/assets"
ln -sf "/usr/share/themes/Graphite-Dark-compact/gtk-4.0/gtk.css" "${HOME}/.config/gtk-4.0/gtk.css"
ln -sf "/usr/share/themes/Graphite-Dark-compact/gtk-4.0/gtk-dark.css" "${HOME}/.config/gtk-4.0/gtk-dark.css"
```
It seems to work...

I need to place the theme as system-wide because I'm thinking the scenario where the theme must be installed at ISO creation, so at a moment where a user does not exist (except for root of course).

**Comment by BrachyKnight** [comment]
Sorry If I barge in this thread. But I've tried using `./install.sh -t orange --tweaks nord rimless -l` but the nautilus themes remains light. Am I using the install script wrong?
![Screenshot from 2022-11-05 15-40-37](https://user-images.githubusercontent.com/47126278/200125275-582b1612-d0a8-435c-a9b2-099b152cd300.png)

**Comment by Asthonishia** [comment]
Same problem for me. Do not work.

**Comment by leoniscsem** [comment]
It seems to be necessary to manually copy the theme over to `/usr/share/icons` and link the `gtk-4.0` files as well in order to have a systemwide consistent experience. E.g. `synaptic` requires to be launched with sudo rights, so it didn't turn out well.

before:
![syanptic-graphite](https://github.com/vinceliuice/Graphite-gtk-theme/assets/6514726/b14628f8-244d-4953-bd80-1607b9274c76)

after:
![synaptic-graphite-after](https://github.com/vinceliuice/Graphite-gtk-theme/assets/6514726/8cab0d89-a7c3-493e-be52-b24d666e9726)
